controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -168,15 +168,13 @@ protected void shutDown() {
 
                 // Initiate stop on all event processor cells and await their termination.
                 for (EventProcessorCell<T> cell : eventProcessorMap.values()) {
-                    log.info("Stopping {}", cell);
-                    // first report reader offline with the last position then shut it down.
-                    readerGroup.readerOffline(cell.getReaderId(), cell.getCheckpoint());
+                    log.info("Stopping event processor cell: {}", cell);
                     cell.stopAsync();
-                    log.info("Awaiting termination of {}", cell);
+                    log.info("Awaiting termination of event processor cell: {}", cell);
                     try {
                         cell.awaitTerminated();
                     } catch (IllegalStateException e) {
```

Comment:
maybe we cab log on all exceptions here and not just illegal state

Reply:
Done.

Reply:
Done.